RoboticonColony/Assets/Editor/Unit_Tests/TileTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using System;
+using UnityEditor;
+using NUnit.Framework;
+using NUnit;
+
+public class TileTest
```

Comment:
We should implement these tests before merging . We should also have some tests for when things go wrong, ie assert errors when someone tries to use AddRoboticon invalidly and so on